### ISSUE TITLE
Form.RadioGroup can't get props.

### ISSUE THREAD

**Issue by eduludi** [zabute/formsy-semantic-ui-react#14]
Looks like is not possible to provide extra params to `Form.RadioGroup` like: 
- `as`
- `inline`
- `grouped`

Since you are using <Form.Group> from `semantic-ui-react` in [`src/FormsyRadioGroup.js:60`](https://github.com/zabute/formsy-semantic-ui-react/blob/master/src/FormsyRadioGroup.js#L60), maybe something like this can solve the issue:

```js
// src/FormsyRadioGroup.js

  render() {
    const {  
      label,
      required,
      formRadioGroup,
      children,
      getValue,
      errorLabel,
      isValid,
      isPristine,
      getErrorMessage,
      ...otherProps, // <== here
    } = this.props;

    return (
      <Form.Group {...otherProps}> // <== and here
        ...
      </Form.Group>
    )
  }
```

Thanks!

**Comment by zabute** [comment]
fixed in ```0.2.9```

**Comment by eduludi** [comment]
Thanks! 👍
